.github/workflows/hashtag-counter.yml

```diff
@@ -0,0 +1,78 @@
+# ------------------------------------------------------------
```

Comment:
Consider merging the 2 yamls if possible as most of the steps are similar

Reply:
They're very similar but they may deploy separately, so I'll keep separate unless this changes.